packages/flutter/lib/src/foundation/basic_types.dart

```diff
@@ -8,6 +8,8 @@ import 'dart:collection';
 // COMMON SIGNATURES
 
 export 'dart:ui' show VoidCallback;
+export 'bitfield.dart'
+  if (dart.html) 'internal_web.dart';
```

Comment:
should this be bitfield_unsupported.dart?

Reply:
....why did this pass the compile test

Reply:
I'd be curious to know as well :)

Reply:
Wow! In that case, I recommend that you `flutter run --release` before merging the PR, in case Dart native AOT is also not handling conditional exports properly.

Reply:
Updating this to the correct import/export did not work as expected